mu/__init__.py

```diff
@@ -1 +1,17 @@
+import gettext
+import locale
+import os
+
+# Configure locale and language
+# Define where the translation assets are to be found.
+localedir = os.path.join('mu', 'locale')
```

Comment:
This needs to be the same as what is currently (in master) in app.py:

```localedir = os.path.abspath(os.path.join(os.path.dirname(__file__), 'locale'))```

Reply:
Thanks; fixed